### ISSUE TITLE
Customize "Username", "Password", "Login"

### ISSUE THREAD

**Issue by paulbricman** [mkhorasani/Streamlit-Authenticator#1]
Hi @mkhorasani, thanks a lot for maintaining this awesome module! I'd like to be able to customize the labels for the two text_inputs and for the button. Specifically, I'd make them lower caps so that they fit in with the rest of the naming pattern in the screenshot below. I could do a PR myself, as I feel there are literally 4 lines of code to change. Let me know what you think!

```python
# current
name, authentication_status = authenticator.login('login', 'sidebar')

# suggestion
name, authentication_status = authenticator.login('login', 'sidebar', 'username', 'password', 'login') # where the new ones have defaults
```

Edit: Same for "Logout" would be nice, too.

![Screenshot from 2022-01-06 10-16-41](https://user-images.githubusercontent.com/20104026/148351598-79190771-bb77-42aa-acef-ac690e5f4bc3.png)

**Comment by mkhorasani** [comment]
Hi Paul,

You're most welcome! Yes, you may modify the lines shown below in the /streamlit_authenticator/__init__.py

![image](https://user-images.githubusercontent.com/38884731/148356822-8da72b62-ca5d-4a20-8dd6-56767a44929d.png)

I will try to add this feature in my next release, so stay tuned.

Cheers.

**Comment by paulbricman** [comment]
Hi @mkhorasani, I got caught up in a few things recently, so I actually don't think I'll be able to look into it myself in the near future.. Sorry for the confusion!

**Comment by PeterKuralt** [comment]
Hi @mkhorasani,

would it be possible to put the logut buttom at the bottom of the page. It looks a bit unatural on the top. Thank you in advance for the answer.

**Comment by mkhorasani** [comment]
Dear all, this issue has now been resolved with the latest release v0.1.3. Thank you all for your patience.
